extractor/src/main/java/com/example/extractor/ReadmeGenerator.java

```diff
@@ -84,7 +84,7 @@ public static void generateIndividualReadme(
             + "```%n"
             + "Make comment in PR:%n"
             + "```%n"
-            + "Github, generate report%n"
+            + "Github, generate report for configs in PR description%n"
```

Comment:
Can we keep both?
Such configs can be used in old workflow  too. Before we fully migrate, let's keep both.